### PR TITLE
Allow `deny` inside `forbid` as a no-op

### PR DESCRIPTION
**Pull request by Noratrieb**
updates the reference to match https://github.com/rust-lang/rust/pull/121560

### CODE REVIEW

**Review by chorman0773 — APPROVED** [review]

**Comment by chorman0773** [comment]
This should wait for 121560 to merge, but otherwise, LGTM.

**Review by ehuss — APPROVED** [review]
Thanks!

**Pull request merged**
